.github/DEV_GUIDE.md

```diff
@@ -361,27 +361,27 @@ You can press the Ticked / Unticked button in the bottom right corner to add / r
 
 ![](https://i.imgur.com/goYeY9R.png)
 
-Do you vaguely remember the name of a command you want to do? Press <kbd>Ctrl + Shift + P</kbd> to open the Command Palette and start typing. There's a decent chance that VS Code will be able to suggest the correct thing. 
+Do you vaguely remember the name of a command you want to do? Press **Ctrl+Shift+P** to open the Command Palette and start typing. There's a decent chance that VS Code will be able to suggest the correct thing. 
 
-One of the tabs in VS Code usually has its name written in italics; that means that it's a temporary tab (often created by Find) and any new tab will replace it. You can turn it into a permanent tab by doubleclicking on it or pressing <kbd>Ctrl + K</kbd> and Enter.
+One of the tabs in VS Code usually has its name written in italics; that means that it's a temporary tab (often created by Find) and any new tab will replace it. You can turn it into a permanent tab by doubleclicking on it or pressing Ctrl + K and Enter.
 
-<kbd>F12</kbd> jumps to the definition of the thing you have your cursor over.
+**F12** jumps to the definition of the thing you have your cursor over.
 
-<kbd>Shift + F12</kbd> shows you all uses of the thing you have your cursor over.
+**Shift+F12** shows you all uses of the thing you have your cursor over.
 
 The above features might stop being accurate if you are doing a lot of changes, to refresh how VS Code sees the environment click the "goonstation" button in the bar at the bottom of the window.
 
 ![](https://i.imgur.com/Lhs7EHK.png)
 
-<kbd>Ctrl + Shift + F</kbd> lets you search across the whole codebase.
+**Ctrl + Shift + F** lets you search across the whole codebase.
 
-Press <kbd>Ctrl + T</kbd> and start typing the path / name of something you want to find, then hit enter to jump to its definition.
+Press **Ctrl+T** and start typing the path / name of something you want to find, then hit enter to jump to its definition.
 
-Press <kbd>Ctrl + K</kbd> and start typing the name of a file you want to open, then hit enter to open it.
+Press **Ctrl+P** and start typing the name of a file you want to open, then hit enter to open it.
 
 ### Using Command Line git
 
-Sometimes it might be useful to type in git commands directly. To do that press <kbd>Ctrl + \`</kbd> to open PowerShell in VS Code. (Or <kbd>Ctrl + Shift + \`</kbd> to make a new PowerShell window.) Though this might depend on your operating system and default shell configuration.
+Sometimes it might be useful to type in git commands directly. To do that press **ctrl + \`** to open PowerShell in VS Code. (Or **ctrl + shift + \`** to make a new PowerShell window.) Though this might depend on your operating system and default shell configuration.
```

Comment:
no. you're reverting my changes to it

Reply:
I don't ever recall touching that part of the dev guide, weird.

Reply:
why request my review when you've not fixed it

Reply:
It would seem my attempt at fixing it earlier didn't actually fix anything, sincere apologies. 

It should be as-is with the master branch now, however

Reply:
Very much not lol? Just look at the changes